transformer_nuggets/llama/train.py

```diff
@@ -82,6 +83,11 @@ class TrainingConfig:
     # on a single batch. It should be set to False for normal training.
     overfit: bool = False
 
+    enable_qlora: bool = False
+    lora_r: int = 8
```

Comment:
Nit: TrainingConfig I think these should maybe go in HyperParams, tried to have logical distitincation between, "effects individual training run" and then hyperparams "effect the model arch" and may want to be searched over

but either way is cool those have already gotten a little blurry